test/models/root_repository_test.rb

```diff
@@ -304,7 +308,7 @@ def test_invalid_upstream_username_update
     end
 
     def test_valid_upstream_authorization
-      ENV['ENCRYPTION_KEY'] = "25d224dd383e92a7e0c82b8bf7c985e9"
+      EncryptionKey.const_set(:ENCRYPTION_KEY, '25d224dd383e92a7e0c82b8bf7c985e9')
```

Comment:
have you thought about trying something like instead of the above patch
```
@root.stubs(:encryption_key).returns("25d224dd383e92a7e0c82b8bf7c985e8")
```